lib/driver.js

```diff
@@ -172,18 +172,12 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.opts.browserName) {
       log.info('Safari test requested');
-      this.safari = true;      
+      this.safari = true;
       this.opts.app = undefined;
       this.opts.processArguments = this.opts.processArguments || {};
-      if (this.isRealDevice()) {
-        this.opts.bundleId = SAFARI_LAUNCHER_BUNDLE;
-        this._currentUrl = this.opts.safariInitialUrl || `http://appium.io`;
-        this.opts.processArguments.args = [this._currentUrl];
-      } else {
-        this.opts.bundleId = SAFARI_BUNDLE_ID;
-        this._currentUrl = this.opts.safariInitialUrl || `http://${this.opts.address}:${this.opts.port}/welcome`;
-        this.opts.processArguments.args = ['-u', this._currentUrl];
-      }
+      this.opts.bundleId = SAFARI_BUNDLE_ID;
+      this._currentUrl = this.opts.safariInitialUrl || DEFAULT_SAFARI_URL;
```

Comment:
@imurchie 

I see the ops.address and opts.port are sent via cap. what happen if the end-use don't set safariInitialUrl and address/port or the url http://${this.opts.address}:${this.opts.port}/welcome doesn't exist

My suggestion we should launch just only the safari browser (or default url: http://appium.io) for both case (real device and simulator)